axelor-base/src/main/resources/domains/Localization.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<domain-models xmlns="http://axelor.com/xml/ns/domain-models"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/domain-models http://axelor.com/xml/ns/domain-models/domain-models_6.1.xsd">
+
+  <module name="base" package="com.axelor.apps.base.db"/>
+
+  <entity name="Localization">
+    <many-to-one name="country" ref="com.axelor.apps.base.db.Country" title="Country"/>
+    <many-to-one name="language" ref="com.axelor.apps.base.db.Language" title="Language"/>
+    <string name="name" title="Name" required="true" search="country,language"/>
+    <!-- <![CDATA[ -->
+    <!-- String name = ""; -->
+    <!-- if (language == null && country != null) { -->
+    <!-- name = country.getName(); -->
+    <!-- } -->
+    <!-- if (language != null && country == null) { -->
+    <!-- name = language.getName(); -->
+    <!-- } -->
+    <!-- if (language != null && country != null) { -->
+    <!-- name = language.getName() + " (" + country.getName() + ")"; -->
+    <!-- } -->
+    <!-- return name; -->
+    <!-- ]]> -->
+    <!-- </string> -->
+
+    <string name="code" title="Code" required="true" search="country,language"/>
+    <!-- <![CDATA[ -->
+    <!-- String code = ""; -->
+    <!-- if (language == null && country != null) { -->
+    <!-- code = country.getAlpha2Code(); -->
+    <!-- } -->
+    <!-- if (language != null && country == null) { -->
+    <!-- code = language.getCode(); -->
+    <!-- } -->
+    <!-- if (language != null && country != null) { -->
+    <!-- code = language.getCode() + " (" + country.getAlpha2Code() + ")"; -->
+    <!-- } -->
+    <!-- return code; -->
+    <!-- ]]> -->
+    <!-- </string> -->
```

Comment:
keep these lines, it's what I want you to do, that way we're sure that the code will always be well formatted.